### ISSUE TITLE
Creating a cohort / multisample VCF

### ISSUE THREAD

**Issue by kvn95ss** [KCCG/mity#20]
Hello,

We have ~100 samples for which we would like to call mitochondrial variants. The cohort contains a mix of related and unrelated samples. We would like to know the recommended way to call variants for all samples in the cohort. 

For ex, a trio was used for test example, so I'm not sure if combining data that way would be feasible for our data. 

Or, would it be better to ```mity call + normalize``` the variants separately for each sample, merging the vcfs, then using ```mity report``` to annotate the variants?

**Comment by kvn95ss** [comment]
UPDATE - I tried to run ```mity call``` with all the bam files, but got this error - 

```mity version 0.3.0
Calling mitochondrial variants
Running FreeBayes in sensitive mode
No variants in VCF with specified chromosome/s
Writing normalised vcf: cohort_output/all_cohort_chrM.mity.vcf.gz
```

And the vcf file was empty with no variants. Looks like merging VCFs might be better option :/

**Comment by kvn95ss** [comment]
I tried to combine the vcfs with ```bcftools merge``` and ran into this error -

```
mity 0.3.0
Generating mity report
Traceback (most recent call last):
  File "/root/scripts/mity", line 29, in <module>
    args.func(args)
  File "/root/scripts/mitylib/commands.py", line 160, in _cmd_report
    report.do_report(args.vcf, args.prefix, args.min_vaf, args.out_folder_path)
  File "/root/scripts/mitylib/report.py", line 494, in do_report
    min_vaf)
  File "/root/scripts/mitylib/report.py", line 180, in make_table
    if float(FORMAT_VAF) > float(min_vaf):
ValueError: could not convert string to float: '.'
```

Any leads appreciated, thanks!

**Comment by drmjc** [comment]
Hi,
I've never tried a huge cohort, as we usually ran mity for probands or trios. So my first recommendation is to run the related samples together in batches, with normalize and report for each family.

If the VCF is empty, my first suggestion is to look at which `--reference` was used and that this matches the MT chromosome name in the BAM, see https://github.com/KCCG/mity#human

The most recent error message looks like a variant in the VCF has '.' for the AD or DP value. This likely arose from running `bcftools merge` in the middle of the toolchain which is not supported. Thanks for the report, we could add some code that checks for this and at least allows the valid variants to be analysed.

**Comment by trentzz** [comment]
Hi, the issues you've mentioned should be resolved in the new release of Mity 1.1.0. If you encounter any further problems, please feel free to open a new issue. Thanks!
